### DISCUSSION TITLE
查看

### DISCUSSION THREAD

**Discussion by lovesL** [tolking/element-pro-components#491] [Q&A]
![image](https://github.com/tolking/element-pro-components/assets/27960630/072a6eb4-8838-4e5c-857c-e9e37396cec7)
点击查看能和编辑的弹出的内容一样吗  然后内容是不可输入。不会是对每个表单使用插槽自定义内容吧 那太麻烦了....

**Comment by tolking** [comment]
自己通过 #crud-detail 插槽自定义
